HashSet.py

```diff
@@ -0,0 +1,45 @@
+// Time Complexity : O(1)
+// Space Complexity :
```

Comment:
Add space complexity, let me know if you have difficulty in identifying the space.